extensions/kusto/src/contextProvider.ts

```diff
@@ -20,7 +20,8 @@ export enum ContextKeys {
 
 const isCloudEditions = [
 	5,
-	6
+	6,
+	11
```

Comment:
Let's take the opportunity to define these as constants instead so it's clear what the editions are.

Reply:
Instead of introducing new constants I've reused existing constants from azdata.DatabaseEngineEdition